process/smartContract/builtInFunctions/saveUserName.go

```diff
@@ -60,7 +58,7 @@ func (s *saveUserName) ProcessBuiltinFunction(
 	if !ok {
 		return nil, process.ErrCallerIsNotTheDNSAddress
 	}
-	if len(vmInput.Arguments) != 1 || len(vmInput.Arguments[0]) != userNameHashLength {
+	if len(vmInput.Arguments) != 1 {
```

Comment:
A maximum length would be good here.

Reply:
No need. We control the DNS SCs which are only allowed to call this